site/docs/master/aws-config.md

```diff
@@ -265,15 +274,17 @@ cat > velero-policy.json <<EOF
     ]
 }
 EOF
-```
-```bash
+    ```
+    ```bash
 aws iam put-role-policy \
```

Comment:
This doesn't render properly in GH, I believe the content needs to be indented to line up with the backticks.